subt/ros/robot/launch/freyja.launch

```diff
@@ -261,4 +263,20 @@
     <param name="sensor_model/miss" value="0.05"/>
   </node>
 
+  <!-- Combining RGBD messages from both cameras into a single stream -->
+  <node pkg="topic_tools" type="relay" name="rgbd_merge_front" args="/rtabmap/rgbd/front/compressed /rtabmap/rgbd/all/compressed"/>
+  <node pkg="topic_tools" type="relay" name="rgbd_merge_rear" args="/rtabmap/rgbd/rear/compressed /rtabmap/rgbd/all/compressed"/>
+  <!-- Merge camera infos into a single stream -->
+  <node pkg="topic_tools" type="relay" name="camera_info_merge_front" args="$(arg robot_name)/rgbd_front/optical/camera_info $(arg robot_name)/rgbd_all/optical/camera_info"/>
+  <node pkg="topic_tools" type="relay" name="camera_info_merge_rear" args="$(arg robot_name)/rgbd_rear/optical/camera_info $(arg robot_name)/rgbd_all/optical/camera_info"/>
+  <!-- Merge both lidars into a single stream. -->
+  <node pkg="topic_tools" type="relay" name="lidar_merge_front" args="$(arg robot_name)/scan_front $(arg robot_name)/scan_all"/>
+  <node pkg="topic_tools" type="relay" name="lidar_merge_rear" args="$(arg robot_name)/scan_rear $(arg robot_name)/scan_all"/>
+
+  <node pkg="proxy" type="traversability" name="local_map" ns="$(arg robot_name)" output="screen">
```

Comment:
This output to screen is also not needed.

Reply:
fixed